RELEASE/scripts/autoscend/auto_king.ash

```diff
@@ -3,112 +3,77 @@ import <autoscend.ash>
-		foreach it in toPull
+	if(get_property("auto_borrowedTimeOnLiberation").to_boolean() && (get_property("_borrowedTimeUsed") == "false"))
+	{
+		if(get_property("_clipartSummons").to_int() < 3)
 		{
-			if(storage_amount(it) > 0)
-			{
-				auto_log_info("Pulling " + storage_amount(it) + " " + it, "green");
-			}
-			pullAll(it);
+			cli_execute("make borrowed time");
 		}
-
-		toPull = $items[Bittycar meatcar, burrowgrub hive, Can of Rain-Doh, Cheap Toaster, chester\'s bag of candy, chroner cross, chroner trigger, the cocktail shaker, Creepy Voodoo Doll, festive warbear bank, glass gnoll eye, infinite BACON machine, picky tweezers, taco dan\'s taco stand flier, Trivial Avocations Board Game, warbear breakfast machine, warbear soda machine];
-		foreach it in toPull
+		if((item_amount($item[Borrowed Time]) > 0) && (my_daycount() > 1))
 		{
-			if(storage_amount(it) > 0)
-			{
-				auto_log_info("Pulling/Using " + storage_amount(it) + " " + it, "green");
-			}
-			pullAndUse(it, 1);
+			use(1, $item[borrowed time]);
 		}
+	}
 
-		pullPVPJunk();
+	visit_url("campground.php?action=workshed");
 
-//		visit_url("lair2.php?preaction=key&whichkey=436");
-		visit_url("museum.php?action=icehouse", false);
-		visit_url("place.php?whichplace=desertbeach&action=db_nukehouse");
-		if((get_property("sidequestOrchardCompleted") != "none") && !get_property("_hippyMeatCollected").to_boolean())
+	if(get_property("auto_snapshot") == "")
+	{
+		if(svn_info("bumcheekascend-snapshot").last_changed_rev > 0)
 		{
-			visit_url("shop.php?whichshop=hippy");
+			cli_execute("snapshot");
 		}
-		visit_url("clan_rumpus.php?action=click&spot=3&furni=3");
-		visit_url("clan_rumpus.php?action=click&spot=3&furni=3");
-		visit_url("clan_rumpus.php?action=click&spot=3&furni=3");
-		visit_url("clan_rumpus.php?action=click&spot=1&furni=4");
-		visit_url("clan_rumpus.php?action=click&spot=4&furni=2");
-		visit_url("clan_rumpus.php?action=click&spot=9&furni=3");
-
-		if(get_property("auto_borrowedTimeOnLiberation").to_boolean() && (get_property("_borrowedTimeUsed") == "false"))
+		if(svn_info("ccascend-snapshot").last_changed_rev > 0)
 		{
-			if(get_property("_clipartSummons").to_int() < 3)
-			{
-				cli_execute("make borrowed time");
-			}
-			if((item_amount($item[Borrowed Time]) > 0) && (my_daycount() > 1))
-			{
-				use(1, $item[borrowed time]);
-			}
-		}
-
-		visit_url("campground.php?action=workshed");
-
-		if(get_property("auto_snapshot") == "")
-		{
-			if(svn_info("bumcheekascend-snapshot").last_changed_rev > 0)
-			{
-				cli_execute("snapshot");
-			}
-			if(svn_info("ccascend-snapshot").last_changed_rev > 0)
-			{
-				cli_execute("cc_snapshot");
-			}
-			set_property("auto_snapshot", "done");
+			cli_execute("cc_snapshot");
 		}
+		set_property("auto_snapshot", "done");
+	}
 
-		visit_url("place.php?whichplace=town_wrong&action=townwrong_precinct");
+	visit_url("place.php?whichplace=town_wrong&action=townwrong_precinct");
```

Comment:
Do we have a similar `visit_url` call to grab your cowboy boots? If not we should add one.

Reply:
The url to visit is `place.php?whichplace=town_right&action=townright_ltt`, btw

Reply:
Yes... actually this is also a workaround for a mafia issue where it fails to update `telegraphOfficeAvailable` when you break prism

Reply:
change made